dvc/params.yaml

```diff
@@ -1,44 +1,3 @@
-datasets:
```

Comment:
Seems fine to remove. Also, notice that not even before this PR (see on branch main) these items were ever used in the dvc.yaml.